bin/restartServices

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+#
+# restartServices
+# Simple wrapper to restart Apache and PHP FPM services; this can be specified in sudoers file as specifically allowed.
+#
+
+set -e
+set -u
+
+service apache2 restart
```

Comment:
If we're going to specify this in the `sudoers` file then we should probably use `sudo` here